src/register.js

```diff
@@ -22,60 +22,35 @@ if (!applicationId) {
   );
 }
 
-/**
- * Register all commands with a specific guild/server. Useful during initial
- * development and testing.
- */
-// eslint-disable-next-line no-unused-vars
-async function registerGuildCommands() {
-  if (!testGuildId) {
-    throw new Error(
-      'The DISCORD_TEST_GUILD_ID environment variable is required.'
-    );
-  }
-  const url = `https://discord.com/api/v10/applications/${applicationId}/guilds/${testGuildId}/commands`;
-  const res = await registerCommands(url);
-  const json = await res.json();
-  console.log(json);
-  json.forEach(async (cmd) => {
-    const response = await fetch(
-      `https://discord.com/api/v10/applications/${applicationId}/guilds/${testGuildId}/commands/${cmd.id}`
-    );
-    if (!response.ok) {
-      console.error(`Problem removing command ${cmd.id}`);
-    }
-  });
-}
-
 /**
  * Register all commands globally.  This can take o(minutes), so wait until
  * you're sure these are the commands you want.
  */
-// eslint-disable-next-line no-unused-vars
-async function registerGlobalCommands() {
-  const url = `https://discord.com/api/v10/applications/${applicationId}/commands`;
-  await registerCommands(url);
-}
-
-async function registerCommands(url) {
-  const response = await fetch(url, {
-    headers: {
-      'Content-Type': 'application/json',
-      Authorization: `Bot ${token}`,
-    },
-    method: 'PUT',
-    body: JSON.stringify([AWW_COMMAND, INVITE_COMMAND]),
-  });
-
-  if (response.ok) {
-    console.log('Registered all commands');
-  } else {
-    console.error('Error registering commands');
-    const text = await response.text();
-    console.error(text);
+const url = `https://discord.com/api/v10/applications/${applicationId}/commands`;
+
+const response = await fetch(url, {
+  headers: {
+    'Content-Type': 'application/json',
+    Authorization: `Bot ${token}`,
+  },
+  method: 'PUT',
+  body: JSON.stringify([AWW_COMMAND, INVITE_COMMAND]),
+});
+
+if (response.ok) {
+  console.log('Registered all commands');
+  const data = await response.json();
+  console.log(JSON.stringify(data, null, 2));
+} else {
+  console.error('Error registering commands');
+  let errorText = `Error registering commands \n ${response.url}: ${response.status} ${response.statusText}`;
+  try {
+    const error = await response.text();
+    if (error) {
+      errorText = `${errorText} \n\n ${error}`;
+    }
+  } catch {
+    // ignore
```

Comment:
should there be logging here at all? not super clear to me when it'd hit this point

Reply:
This would only happen if you have a response that contains no body, or a super weird body where `response.text()` somehow throws.  It's ultra-safeguarding so we don't throw while trying to parse an error message. That having been said, adding a little logging :)